data_steward/utils/sandbox.py

```diff
@@ -63,3 +89,76 @@ def check_and_create_sandbox_dataset(project_id, dataset_id):
     if sandbox_dataset not in datasets:
         create_sandbox_dataset(project_id, dataset_id)
     return sandbox_dataset
+
+
+def get_sandbox_labels_string(src_dataset_name,
+                              class_name,
+                              table_tag,
+                              shared_lookup=False):
+    """A helper function that formats a set of labels for BigQuery
+
+    :param str src_dataset_name: A dataset name
+    :param str class_name: A class name
+    :param str table_tag: A table tag
+    :param bool shared_lookup: A boolean that indicates if a table is a shared lookup, defaults to False
+    :raises ValueError: Raises if one of first three arguments is whitespace or None
+    :raises ValueError: Raises if the `shared_lookup` is not a boolean
+    :return: A formatted string of the arguments concatenated as labels
+    :rtype: str
+    """
+    STRING_LABELS = ['src_dataset', 'class_name', 'table_tag']
+    BOOL_LABELS = ['shared_lookup']
+
+    labels = OrderedDict([('src_dataset', src_dataset_name),
+                          ('class_name', class_name), ('table_tag', table_tag),
+                          ('shared_lookup', shared_lookup)])
+
+    for label in STRING_LABELS:
+        value = labels[label]
+        if not value or value.isspace():
+            raise ValueError(
+                f"Label '{label}' requires a non-empty string. Received the value '{value}'."
+            )
+
+    for label in BOOL_LABELS:
+        value = labels[label]
+
+        if type(value) != bool:
+            raise ValueError(
+                f"Label '{label}' requires a boolean value of True or False. Received the value '{value}'."
+            )
+
+    return TABLE_LABELS_STRING.render(labels=labels)
+
+
+def get_sandbox_table_description_string(description):
+    """A helper function that returns a formatted description for BigQuery
+
+    :param str description: A table description
+    :return: A formatted table description
+    :rtype: str
+    """
```

Comment:
Do you know if this is required? Not sure if there should be a check in this function that something was actually passed?

Reply:
I'm not sure. In the task, it doesn't specifically mention a required check for description. But I can add it if you think that's best.